homeassistant/components/media_player/snapcast.py

```diff
@@ -107,12 +107,14 @@ def state(self):
     @property
     def unique_id(self):
         """Return the ID of snapcast group."""
-        return '{}{}'.format(GROUP_PREFIX, self._group.identifier)
+        return '{}{}_{}'.format(GROUP_PREFIX, self._group._server._host,
+                                self._group.identifier)
 
     @property
     def name(self):
         """Return the name of the device."""
-        return '{}{}'.format(GROUP_PREFIX, self._group.identifier)
+        return '{}{}_{}'.format(GROUP_PREFIX, self._group._server._host,
```

Comment:
Changing the name will change the entity_id, so that will make this a breaking change.

With the unique id set, the entity_id doesn't need to be changed. Both name and entity_id can be customized by the user.

Reply:
Hmm, I see. Reason for changing it was the same name twice in the tab.
So should I keep the previous name and make renaming inside the registry file?

Reply:
That's my suggestion. I'm guessing not all users will need to change this if they don't have multiple hosts. So it's better to not make a breaking change and still allow impacted users to set their names and IDs as they want.

Reply:
ok I changed it :)